CHANGELOG.md

```diff
@@ -4,6 +4,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [1.20.3] - 2020-04-27
```

Comment:
This is a breaking change, I think we should make it more higher version. If you remember, we even talked about upgrading it to version 2.0.0. Please double check with @erdoganoksuz and @cihadhoruzoglu for more detailed information. I am not very sure about this.